get_scores.py

```diff
@@ -19,16 +38,20 @@ def lambda_handler(event, context):
     body = {}
     with db_session(os.environ['HOST'], context.parameters['/prod/camelot/db-password']) as session:
         # Setup players map, and give each player a starting score of 0 in case they haven't played any games
-        players = {player.id:{'id': player.id, 'name': player.full_name(), 'score':0} for player in session.query(Player).all()}
+        players = {player.id:{'id': player.id, 'name': player.full_name(), 'score':0, 'win':0, 'loss':0} for player in session.query(Player).all()}
         # Handle points from each game
         for game in session.query(Game).all():
             # Handle MVP and LVP points for this game
-            add_point(players, game.mvp_id)
-            subtract_point(players, game.lvp_id)
+            add_mvp(players, game.mvp_id)
+            add_lvp(players, game.lvp_id)
             # Handle all the winners for this game
             for gamePlayer in game.game_players:
                 if gamePlayer.role.is_good() == game.outcome.is_good():
-                    add_point(players, gamePlayer.player_id)
+                    add_win(players, gamePlayer.player_id)
+                else:
+                    add_loss(players, gamePlayer.player_id)
+                update_totals(players, gamePlayer.player_id)
```

Comment:
Seems like we are calling this too many times. Doubt it really matters performance wise since our data set will be so small, but would it make more sense to call it in a final loop over all players?